app/assets/javascripts/app/azlist.js

```diff
@@ -0,0 +1,27 @@
+
+var oldText = "All Facilities"
+
+function removeExtraClasses(classlist) {
```

Comment:
Can we wrap the majority of this file in `$(function() { <code here> });` That way these methods aren't global and subject to naming conflicts?